src/components/notifications/noti-buttons.css

```diff
@@ -0,0 +1,100 @@
+oui-noti-buttons > button.oui-noti-buttons__unread,
+oui-noti-buttons > button.oui-noti-buttons__read,
+oui-noti-buttons > button.oui-noti-buttons__unused {
+  transition: all 0.05s;
```

Comment:
Probably you do not need the `button` tag selector since the class is very specific.